client/components/cart.js

```diff
@@ -1,96 +1,85 @@
-import React from 'react'
+import React, {useEffect} from 'react'
 import {
   fetchItems,
-  addAnItem,
+  addAnItem, // do we need this as part of the cart?
```

Comment:
If we had time to implement it, I wanted to have a 'buy similar items' bar on the cart page where the user can click on the item and automatically add it to the cart.